src/test/resources/MtAmazonDynamoDbChainTest-expectedLogMessages.txt

```diff
@@ -1,5 +1,5 @@
-method=putItem(), table=ctx1.mt_sharedtablestatic_s_nolsi, item={hk={S: ctx1.Table1.1,}, someField={S: someValue1,}}
-method=putItem(), table=ctx2.mt_sharedtablestatic_s_nolsi, item={hk={S: ctx2.Table1.1,}, someField={S: someValue2,}}
+method=putItem(), table=ctx1.mt_sharedtablestatic_s_nolsi, , item={hk={S: ctx1.Table1.1,}, someField={S: someValue1,}}
```

Comment:
see earlier comment about ","

Reply:
fixed in 6fb6604ff5996f2b174b052fb1b38d45fbc9ab63